packages/core/src/loaders/decimatedVolumeModifiers/inPlaneDecimationModifier.ts

```diff
@@ -0,0 +1,51 @@
+import type Point3 from '../../types/Point3';
+import type { ImageVolumeProps } from '../../types';
+import type {
+  DecimatedVolumeModifier,
+  DecimatedVolumeModifierContext,
+} from './types';
+
+export const inPlaneDecimationModifier: DecimatedVolumeModifier = {
```

Comment:
Please add comments about what this modifier is doing.  I can figure it out from the code, but the name isn't quite descriptive enough for me to understand it without reading the code.